--- /home/mockbuild/zabbix.spec	2016-02-28 10:17:00.000000000 +0300
+++ /home/mockbuild/zabbix3.spec	2016-03-10 11:58:48.000000000 +0300
@@ -1,13 +1,11 @@
 Name:		zabbix
 Version:	3.0.1
-Release: 	1%{?alphatag:.%{alphatag}}%{?dist}
+Release: 	2%{?alphatag:.%{alphatag}}%{?dist}
 Summary:	The Enterprise-class open source monitoring solution
 Group:		Applications/Internet
 License:	GPLv2+
 URL:		http://www.zabbix.com/
 Source0:	zabbix-%{version}%{?alphatag:%{alphatag}}.tar.gz
-Source1:	zabbix-web22.conf
-Source2:	zabbix-web24.conf
 Source3:	zabbix-logrotate.in
 Source4:	zabbix-java-gateway.init
 Source5:	zabbix-agent.init
@@ -19,6 +17,7 @@
 Source13:	zabbix-java-gateway.service
 Source14:	zabbix_java_gateway-sysd
 Source15:	zabbix-tmpfiles.conf
+Source16:       zabbix-nginx.conf
 Patch0:		config.patch
 Patch1:		fonts-config.patch
 Patch2:		fping3-sourceip-option.patch
@@ -227,8 +226,8 @@
 Summary:			Zabbix web frontend common package
 Group:				Application/Internet
 BuildArch:			noarch
-Requires:			httpd
-Requires:			php >= 5.4
+Requires:			nginx
+Requires:			php-fpm >= 5.4
 Requires:			php-gd
 Requires:			php-bcmath
 Requires:			php-mbstring
@@ -413,12 +412,8 @@
 mv $RPM_BUILD_ROOT%{_datadir}/zabbix/conf/maintenance.inc.php $RPM_BUILD_ROOT%{_sysconfdir}/zabbix/web/
 
 # drop config files in place
-mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/httpd/conf.d
-%if 0%{?rhel} >= 7
-install -m 0644 -p %{SOURCE2} $RPM_BUILD_ROOT%{_sysconfdir}/httpd/conf.d/zabbix.conf
-%else
-install -m 0644 -p %{SOURCE1} $RPM_BUILD_ROOT%{_sysconfdir}/httpd/conf.d/zabbix.conf
-%endif
+mkdir -p $RPM_BUILD_ROOT%{_sysconfdir}/nginx/conf.d
+install -m 0644 -p %{SOURCE16} $RPM_BUILD_ROOT%{_sysconfdir}/nginx/conf.d/zabbix.conf
 %endif
 
 # install configuration files
@@ -982,10 +977,10 @@
 %files web
 %defattr(-,root,root,-)
 %doc AUTHORS ChangeLog COPYING NEWS README
-%dir %attr(0750,apache,apache) %{_sysconfdir}/zabbix/web
-%ghost %attr(0644,apache,apache) %config(noreplace) %{_sysconfdir}/zabbix/web/zabbix.conf.php
+%dir %attr(0750,nginx,nginx) %{_sysconfdir}/zabbix/web
+%ghost %attr(0644,nginx,nginx) %config(noreplace) %{_sysconfdir}/zabbix/web/zabbix.conf.php
 %config(noreplace) %{_sysconfdir}/zabbix/web/maintenance.inc.php
-%config(noreplace) %{_sysconfdir}/httpd/conf.d/zabbix.conf
+%config(noreplace) %{_sysconfdir}/nginx/conf.d/zabbix.conf
 %{_datadir}/zabbix
 
 %files web-mysql
@@ -1000,6 +995,11 @@
 
 
 %changelog
+* Thu Mar 10 2016 Aleksandr Chernyshev <wmlex@yandex.ru> - 3.0.1-2
+- deleted depending httpd, php
+- added dependency nginx, php-fpm
+- added config to nginx
+
 * Sun Feb 28 2016 Kodai Terashima <kodai.terashima@zabbix.com> - 3.0.1-1
 - update to 3.0.1
 - remove DBSocker parameter
